templates/zerver/help/zulip-glossary.md

```diff
@@ -49,6 +49,13 @@ phrase (a complete list of emojis can be found
 the **smiley face** (<i class="icon-vector-smile"></i>) icon below the
 [compose box](#compose-box).
 
+### emoji reaction
+
+**Emoji reactions** are small [emoji](#emoji) icons that appear below messages.
+Although they are generally used to show reactions to a message, they can be
+used for various other purposes, such as voting or checking off items in a
```

Comment:
Since the glossary is strictly text-based so far, I added external links to the corresponding images in the glossary.